articles/tech-resources/deprecated-extended-error-codes.md

```diff
@@ -0,0 +1,3103 @@
+            }
+
++ Response AB Test not enabled
+
+    + Current 400 (application/json)
+
+            {
+                "errors": [
+                    {
+                        "message": "invalid uri",
+                        "description": "A/B Testing has been disabled",
+                        "code": "1101"
+                    }
+                ]
+            }
+
+    + Post Change Response 400 (application/json)
+
+            {
+                "errors" : [
+                    {
+                        "message": "A/B Testing has been disabled",
+                    }
+                ]
+            }     
+
++ Response Invalid UTF8 found in json
+
+    + Current 400 (application/json)
+
+            {
+                "errors": [
+                    {
+                        "message": "invalid UTF-8",
+                        "description": "Non-UTF-8 data found in json",
+                        "code": "1305"
+                    }
+                ]
+            }
+
++ Reponse already in use
+
+    + Current 409 (application/json)
+
+            {
+                "errors": [
+                    {
+                        "message": "resource conflict",
+                        "description": "Template is in use by msg generation",
+                        "code": "1602"
+                    }
+                ]
+            }
+
+    + Post Change Response 409 (application/json)
+
+            {
+                "errors" : [
+                    {
+                        "message": "Template is in use by msg generation",
+                    }
+                ]
+            }
+
++ Response missing template id or content field
+
+    + Current 422 (application/json)
+
+            {
+                "errors": [
+                    {
+                        "message": "required field is missing",
+                        "description": "content object or template_id required",
+                        "code": "1400"
+                    }
+                ]
+            }
+
+    + Post Change Response 422 (application/json)
+
+            {
+                "errors" : [
+                    {
+                        "message": "content object or template_id required",
+                    }
+                ]
+            } 
+
++ Response Template does not exist
+
+    + Current 422 (application/json)
+
+            {
+                "errors": [
+                    {
+                        "message": "Subresource not found",
+                        "description": "template 'nonexistent' does not exist",
+                        "code": "1603"
+                    }
+                ]
+            }
+
+    + Post Change Response 422 (application/json)
+
+            {
+                "errors" : [
+                    {
+                        "message": "template '%nonexistent%' does not exist",
+                    }
+                ]
+            } 
+
++ Reponse not specify template_id and ab_test_id

```

Comment:
> Note: This will be returned out of band for the Transmission API

{
                "errors": [
                    {
                        "message": "Error in content.email_rfc822: %errorMsg%",
                    }
                ]
            }